mathics/core/expression.py

```diff
@@ -734,6 +734,8 @@ def evaluate(self, evaluation) -> typing.Union["Expression", "Symbol"]:
 
         old_options = evaluation.options
         evaluation.inc_recursion_depth()
+        if evaluation.show_steps:
+            print("\t Evaluating: ", expr)
```

Comment:
@mmatera this works only with Mathics CLI and mathics-script, not Mathics Django. I think it is better to use `evaluation.message`.

Reply:
My idea was to send it to a terminal, just for doing debugging tasks. But if you think it is useful in the graphics interface, go ahead...